webserver.js

```diff
@@ -0,0 +1,8 @@
+var http = require("http");
+var server = http.createServer(function(request, response) {  
+  response.writeHead(200, {"Content-Type": "text/html"});  
+  response.write("<!DOCTYPE "html">");  
```

Comment:
"html" will not run because of quote mismatch. Need to change it to single quotes